.gitignore

```diff
@@ -4,3 +4,140 @@
 /.idea/libraries
 .DS_Store
 /build
+
```

Comment:
Are you thinking of the lines I have added? Those are taken from Github’s official Python .gitignore (https://github.com/github/gitignore/blob/master/Python.gitignore) and Jetbrains .gitignore files (https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore).